deepcell/model_zoo/panopticnet.py

```diff
@@ -236,6 +168,10 @@ def __create_semantic_head(pyramid_dict,
     Returns:
         keras.layers.Layer: The semantic segmentation head
     """
+
+    conv = Conv2D if ndim == 2 else Conv3D
+    conv_kernel = (1, 1) if ndim == 2 else (1, 1, 1)

```

Comment:
Could make this cooler like `conv_kernel = (1,) * ndim`
---
assuming `ndim` is either 2 or 3.

Reply:
The only downside to that change is that we could end up with a 4 or 5d kernel if the user submitted the wrong value for ndim. Perhaps the most appropriate change would be to first check that ndim is 2 or 3?